components/Stock/YarnStockForm/index.js

```diff
@@ -24,88 +34,193 @@ export default function YarnStockForm({ setAddYarnStockStatus }) {
-      </YarnForm>
+            radius="1rem"
+            name="gramm"
+            backgroundColor="#f5f5f5"
+            defaultValue={editedYarnStock.gramm}
+          />
+          <label htmlFor="meter">Length per skein</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="meter"
+            backgroundColor="#f5f5f5"
+            defaultValue={editedYarnStock.meter}
+          />
+          <label htmlFor="color">Color</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="color"
+            backgroundColor="#f5f5f5"
+            defaultValue={editedYarnStock.color}
+          />
+          <label htmlFor="skein">Skein</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="skein"
+            backgroundColor="#f5f5f5"
+            defaultValue={editedYarnStock.skein}
+          />
+
+          <ButtonContainer>
+            <StyledButton
+              width="8rem"
+              height="3rem"
+              onClick={handleCancelAddYarnStock}
+            >
+              Cancel
+            </StyledButton>
+            <StyledButton width="8rem" height="3rem">
+              Confirm
+            </StyledButton>
+          </ButtonContainer>
+        </YarnForm>
+      ) : (
+        <YarnForm onSubmit={(event) => handleAddYarnStockSubmit(event)}>
+          <Upload setYarnImageUrl={setYarnImageUrl} />
+          <label htmlFor="brand">Brand</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="brand"
+            backgroundColor="#f5f5f5"
+          />
+          <label htmlFor="type">Type</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="type"
+            backgroundColor="#f5f5f5"
+          />
+          <label htmlFor="gramm">Weight per skein</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="gramm"
+            backgroundColor="#f5f5f5"
+          />
+          <label htmlFor="meter">Length per skein</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="meter"
+            backgroundColor="#f5f5f5"
+          />
+          <label htmlFor="color">Color</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="color"
+            backgroundColor="#f5f5f5"
+          />
+          <label htmlFor="skein">Skein</label>
+          <StyledInput
+            width="15rem"
+            height="3rem"
+            radius="1rem"
+            name="skein"
+            backgroundColor="#f5f5f5"
+          />
```

Comment:
Also here you don't have to give every element the same properties.
You could use the `StyledInput` in another styled component to give it other dimensions:

const YarnFormInput = styled(StyledInput)
  width="15rem";
  height="3rem";
  radius="1rem";

And then use `YarnFormInput`everywhere you use `StyledInput`